### DISCUSSION TITLE
Importing .tumblr files

### DISCUSSION THREAD

**Discussion by 0x44404** [TumblThreeApp/TumblThree#334] [Q&A (Question & Answer)]
Hello !
I've used this software since years, but i recently got a problem with drive making me loose my blog list. But i still have in the index folder all of the .tumblr files. Is ther any way to import them in TumblThree ?
Thanks in advance.

**Comment by thomas694** [comment]
Hello,
maybe I didn't understand well what you mean by "loosing your blog list". But if you still have the contents of the index folder you can either copy these files to your new installation's index folder, or you can go to the settings dialog and change your "download location" to your old index folder's parent folder. In both cases, you should have your blogs listed again after starting TumblThree.

**Comment by 0x44404** [comment]
oh shit, indeed my bad im just dumb as shit. Thanks a lot mate !
